table/server/master/src/main/java/alluxio/master/table/Table.java

```diff
@@ -107,7 +107,7 @@ public boolean sync(UdbTable udbTable) {
         changed = true;
       }
       List<Partition> partitions = mPartitionScheme == null
-          ? new ArrayList<>() : mPartitionScheme.getPartitions();
+          ? new ArrayList<>() : new ArrayList<>(mPartitionScheme.getPartitions());
```

Comment:
is `new ArrayList<>()` allowed to be `Collections.emtpyList()`?

Reply:
nope, this list needs to be mutable